ksqldb-parser/src/main/java/io/confluent/ksql/parser/tree/CreateStream.java

```diff
@@ -83,4 +88,25 @@ public String toString() {
         .add("properties", getProperties())
         .toString();
   }
+
+  private static void throwOnPrimaryKeys(final TableElements elements) {
+    final String badEs = elements.stream()
+        .filter(e -> e.getNamespace().isKey() && e.getNamespace() != Namespace.KEY)
+        .map(badE ->
+            badE.getLocation().map(NodeLocation::asPrefix).orElse("") + badE.getName())
+        .collect(Collectors.joining(System.lineSeparator()));
+
+    if (!badEs.isEmpty()) {
+      throw new KsqlException("The following columns are defined as PRIMARY KEY columns. "
+          + "Streams do not support PRIMARY KEY columns, only KEY columns."
```

Comment:
nit: we should start with the error message (`Streams do not support ...`) and then give what failed (thinking about cases where only number of chars are returned)

Reply:
done